spec/features/users/user_nav_bar_spec.rb

```diff
@@ -89,24 +85,25 @@
     end
   end
 
-  # describe "as a merchant employee" do
-  #   it "does not allow me to access any path that begins with '/admin' and displays a 404 error" do
-  #     merchant = User.create!(name: "Josh Tukman",
-  #                           address: "756 Main St",
-  #                           city: "Denver",
-  #                           state: "Colorado",
-  #                           zip: "80209",
-  #                           email: "josh.t@gmail.com",
-  #                           password: "secret_password",
-  #                           password_confirmation: "secret_password",
-  #                           role: 1)
-  #
-  #   allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(merchant)
-  #
-  #   visit "/admin"
-  #
-  #   expect(page).to have_content("The page you were looking for doesn't exist.")
-  #   end
+  describe "as a merchant employee" do
```

Comment:
Was this test turned off?  Good catch turning it back on if so!

Reply:
Yes! There were a couple that were turned off that I turned back on.